pkg/controller/siddhiprocess/artifacts_test.go

```diff
@@ -189,7 +189,7 @@ func TestCreateDeployment(t *testing.T) {
 		configs.SiddhiImage,
 		"siddhirunner",
 		[]string{configs.Shell},
-		[]string{configs.SiddhiHome + configs.RunnerRPath},
+		[]string{configs.SiddhiHome + configs.SiddhiBin + "/" + configs.SiddhiProfile + ".sh"},
```

Comment:
You can use `filepath` package to generate paths. https://golang.org/pkg/path/filepath/

Reply:
Fixed in https://github.com/siddhi-io/siddhi-operator/pull/123/commits/785753fcea0dd5dcef62c63fe739e8e53b54e03e